spec/integration/posts_index_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'User posts', type: :system, js: true do
```

Comment:
- [ ] Please add a test that asserts you have a pagination button on this page as per the requirement. The button doesn't need to be functional at this point. The key point is that button is only visible ' if there are more posts than fit on the view.'

Reply:
Thank you for the feedback, @tugsem. We'll implement the changes shortly.